google-cloud-firestore/src/test/java/com/google/cloud/firestore/LocalFirestoreHelper.java

```diff
@@ -929,14 +928,30 @@ public static String autoId() {
     return fromJsonString(json.replace("'", "\""));
   }
 
-  /** Map of request/response pairs to be returned when `sendRequest()` is called. */
-  static class RequestResponseStubber
+  /**
+   * Contains a map of request/response pairs that are used to create stub responses when
+   * `sendRequest()` is called.
+   */
+  static class ResponseStubber
       extends LinkedHashMap<GeneratedMessageV3, ApiFuture<? extends GeneratedMessageV3>> {
+    ApiFuture<? extends GeneratedMessageV3> checkOneActiveRequestAndReturnResponse(
```

Comment:
This method should:
- have a generic name
- be document
- be overridden in the SerialResponseStubber and perform a "checkOneActiveRequestAndReturnResponse".

Reply:
(ignore the previous version of this comment)

Reply:
Renamed to `verifyResponse` and added comments.